statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -817,7 +866,7 @@ def mean(self):
 
         .. math:: \mu = (I - A_1 - \dots - A_p)^{-1} \alpha
         """
-        return np.linalg.solve(self._char_mat, self.exog)
+        return np.linalg.solve(self._char_mat, self.intercept)
```

Comment:
Possibly a warning that this method is only correct in the no-exog case?

Reply:
I will change the docstring to long run intercept.
I'm not sure yet about deprecating this method.